src/Mailer/TraceableMailer.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Contributte\Mail\Mailer;
+
+use Contributte\Mail\Tracy\MailPanel;
+use Nette\Mail\IMailer;
+use Nette\Mail\Message;
+
+final class TraceableMailer implements IMailer
+{
+
+	/** @var IMailer|null */
+	private $mailer;
+
+	/** @var MailPanel */
+	private $panel;
+
+	/**
+	 * @param IMailer $mailer
+	 * @param MailPanel $panel
+	 */
+	public function __construct(Imailer $mailer, MailPanel $panel)
```

Comment:
I would prefer to store mails in array and reverse the dependency on MailPanel. 

Because: this has just one usage, show emails in panel

If MailPanel will have dependency on TraceableMailer, then TraceableMailer still stores emails and more other classes could retrieve them.

Reply:
Yeah sure. I did it this way because I wanted to be able to use MailPanel without dependency on anything...just to show config and to have an option to display sent email if I wanted. But sure I can no problem rework it this way it is more convenient.